compiler/rustc_feature/src/active.rs

```diff
@@ -684,6 +684,9 @@ declare_features! (
     /// Allows unnamed fields of struct and union type
     (incomplete, unnamed_fields, "1.53.0", Some(49804), None),
 
+    /// Allows converting enums to/from their repr.
+    (active, enum_as_repr, "1.54.0", None, None),
```

Comment:
To do before merging: How do tracking issues work?

Reply:
You file an issue using the tracking issue template, and then use that issue number here.